test/testUtils/mtohUtils.py

```diff
@@ -39,6 +39,9 @@ def checkForPlugin(pluginName: str):
 def checkForMayaUsdPlugin():
     return checkForPlugin('mayaUsdPlugin')
 
+def checkForMtoAPlugin():
+    return checkForPlugin('mtoa')
```

Comment:
Thanks for the updates. Does this checkForPlugin loads the plugin or just checks if it is loaded ?